master/docs/manual/cfg-schedulers.rst

```diff
@@ -811,22 +811,27 @@ What you need in your config file is something like::
                  name="force",
                  builderNames=["my-builder"],
 
-                 # will generate a combo box
-                 branch=util.ChoiceStringParameter(name="branch",
-                                                   choices=["main","devel"],
-                                                   default="main"),
+                 codebases = [ util.CodebaseParameter(
```

Comment:
I think there (and below) shouldn't be spaces around "=" and after "[" according to PEP8